Automattic/newspack-theme
# Hide slideout sidebars when printing

### All Submissions:


### Changes proposed in this Pull Request:

This fixes an odd issue when printing a non-AMP version of a Newspack site using Safari.

See 1203981098043663-as-1203979873000356

### How to test the changes in this Pull Request:

1. Navigate to Customizer > Header Settings > Slide-out sidebar, and enable the slide-out sidebar.
2. Navigate to Customizer > Widgets > Slide-out sidebar, and add at least one widget to that space. 
3. Disable AMP, or switch to AMP transitional.
4. View a single post in Safari and print - note that in the print preview, you get a tiny little sliver of the post:

![image](https://user-images.githubusercontent.com/177561/222850612-d93a3871-ed43-4752-b708-b38cd48eacd8.png)

5. Apply the PR and run `npm run build`. 
6. Confirm that the print preview now shows the whole post:

![image](https://user-images.githubusercontent.com/177561/222850806-dd1dad93-77bc-4b06-b16e-d0203e3981f3.png)

### Other information:
